spec/gp_update_metadata_source_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+
+describe Fastlane::Actions::GpUpdateMetadataSourceAction do
+  it 'updates any block in a given .po file with the values from the given sources' do
+    Dir.mktmpdir do |dir|
+      # 1: Create a dummy .po file to use as input.
+      output_path = File.join(dir, 'output.po')
+      dummy_text = <<~PO
+        msgctxt "v1.0-whats-new"
+        msgid "this value should change"
+        msgstr ""
+        msgctxt "release_note_0122"
```

Comment:
Would be nice to also have `release_note_0121` in here in addition to this one, so that your test also demonstrates/checks that the N-2 notes get **removed** from the `.po` once release notes for N get added.

Reply:
Good point. Will do.